trl/trainer/sft_trainer.py

```diff
@@ -422,7 +422,7 @@ def concat_prompt_completion(example):
                     map_kwargs["desc"] = f"Tokenizing {dataset_name} dataset"
 
                 def tokenize(example, processing_class, dataset_text_field):
-                    return processing_class(example[dataset_text_field])
+                    return processing_class(text=example[dataset_text_field])
```

Comment:
ah yes good catch! i forgot this one

Reply:
yes I believe so... since the Gemma3 processor has images as the first argument and text as the 2nd and its being called without any arguments..